libmultilabel/data_utils.py

```diff
@@ -140,7 +158,7 @@ def load_or_build_text_dict(
 
     if os.path.exists(embed_file):
         logging.info(f'Load pretrained embedding from file: {embed_file}.')
-        embedding_weights = get_embedding_weights_from_file(vocabs, embed_file, silent)
+        embedding_weights = get_embedding_weights_from_file(vocabs, embed_file, silent, normalize)
```

Comment:
so the "normalize" option is only for user-specified embeddings?  If I am a user I would think that this is applied to all situations. Need some discussion tomorrow